packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/modelservice/ModelServiceProxy.java

```diff
@@ -0,0 +1,58 @@
+package software.uncharted.terarium.hmiserver.proxies.modelservice;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import software.uncharted.terarium.hmiserver.models.modelservice.Graph;
+import software.uncharted.terarium.hmiserver.models.modelservice.SimulateParams;
+import software.uncharted.terarium.hmiserver.models.modelservice.ModelCompositionParams;
+
+@RegisterRestClient(configKey = "model-service")
+@Produces(MediaType.APPLICATION_JSON)
+public interface ModelServiceProxy {
+	@PUT
+	@Path("api/models")
```

Comment:
Isn't bit confusing to use `api` in the url when this service will do nothing else?

Reply:
You wanted the model-service to encapsulate all modelling operations, so eventually it will be something like
- /api/models
- /api/simulations 
- ...

Reply:
Sure, my points was just because the `model-service` will only have an API, why not just be:
* /models
* /simulations
* ...